### PR TITLE
MRG: Allow to specify custom landmark names in get_head_mri_trans()

### PR DESCRIPTION
**Pull request by hoechenberger**
See https://github.com/mne-tools/mne-python/issues/10208#issuecomment-1014880772 is for why this is useful.

Merge checklist
---------------

Maintainer, please confirm the following before merging:

- [ ] All comments resolved
- [ ] This is not your own PR
- [x] All CIs are happy
- [x] PR title starts with [MRG]
- [x] [whats_new.rst](https://github.com/mne-tools/mne-bids/blob/main/doc/whats_new.rst) is updated
- [ ] PR description includes phrase "closes <#issue-number>"

### CODE REVIEW

**Comment by hoechenberger** [comment]
This is ready for review. I'm not particularly proud of that code, but it does do the job!

File: doc/whats_new.rst [hunk 1]
```diff
@@ -40,6 +40,8 @@ Enhancements
 
 - Add support for CNT (Neuroscan) files in :func:`mne_bids.write_raw_bids`, by `Yorguin Mantilla`_ (:gh:`924`)
 
+:func:`mne_bids.get_head_mri_trans` gained a new parameter, ``landmark_names``, allowing to specify custom names used for the anatomical landmarks in the MRI data, by `Richard Höchenberger`_ (:gh:`xxx`)
```
Comment (agramfort) [inline doc/whats_new.rst:43]: fix xxx

**Review by agramfort — COMMENTED** [review]
I am bit worried of "custom names" in a standard...

**Comment by hoechenberger** [comment]
> I am bit worried of "custom names" in a standard...

I hear you and +10000 this.

Well, what can we do, we ain't gonna fix this anytime soon :\
It's a mess.

File: doc/whats_new.rst (hunk 1 above)
Comment (hoechenberger) [inline doc/whats_new.rst:43]:
```suggestion
:func:`mne_bids.get_head_mri_trans` gained a new parameter, ``landmark_names``, allowing to specify custom names used for the anatomical landmarks in the MRI data, by `Richard Höchenberger`_ (:gh:`946`)
```

**Comment by hoechenberger** [comment]
Superseded by #955

**Pull request closed without merging**